Formula/amap.rb

```diff
@@ -20,6 +20,8 @@ class Amap < Formula
     sha256 "1361e89caf2590146c5872907f90ad67ac9b99d2198320691e9f6df0cfdbe16c" => :sierra
   end
 
+  disable! date: "2018-06-04" because: :unmaintained
```

Comment:
missing a comma

Reply:
Argh, will fix.

Sidenote: Had to submit this without my usual check, because `brew audit` just bails on disabled formulae.

Reply:
Note that choosing this disable date is basically equivalent to just removing the formula now. I'd choose a date closer to today.

Reply:
Okay, let's do the release date of Big Sur